src/App.vue

```diff
@@ -85,20 +88,20 @@ export default {
 				this.setActiveTableId(parseInt(currentRoute.params.tableId))
 				this.setPageTitle(this.activeTable.title)
 				if (!currentRoute.path.includes('/row/')) {
-					this.switchActiveMenuEntry(document.querySelector('header .header-left .app-menu a[title="Tables"]'))
+					this.switchActiveMenuEntry(document.querySelector(`header .header-start .app-menu a[href="${url}"]`))
```

Comment:
If you have any other idea how to make it works for non-en locales - feel free to suggest

Reply:
see https://github.com/nextcloud/server/pull/51207

It can looks like:

```
if (currentRoute.path.startsWith('/table/')) {
	this.setActiveTableId(parseInt(currentRoute.params.tableId))
	this.setPageTitle(this.activeTable.title)
	if (!currentRoute.path.includes('/row/')) {
		emit('nextcloud:app-menu.active', 'tables')
}
```